packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -403,9 +411,11 @@ class Auth {
         this.set('customer_type', isGuest ? 'guest' : 'registered')
 
         const refreshTokenKey = isGuest ? 'refresh_token_guest' : 'refresh_token_registered'
-        const expiresDate = res.refresh_token_expires_in
-            ? this.convertSecondsToDate(res.refresh_token_expires_in)
-            : undefined
+        const fallbackTTL =
+            typeof this.refreshTokenTTL === 'number' && this.refreshTokenTTL >= 0
+                ? this?.refreshTokenTTL
+                : res.refresh_token_expires_in || undefined
```

Comment:
nit, do we need the last part `|| undefined`? is `refresh_token_expires_in` conditionally defined from the API response and sometimes not available?

```suggestion
                : res.refresh_token_expires_in
```

Reply:
Done ✅